lib/private/DB/QueryBuilder/FunctionBuilder/SqliteFunctionBuilder.php

```diff
@@ -31,6 +31,11 @@ public function concat($x, $y): IQueryFunction {
 		return new QueryFunction('(' . $this->helper->quoteColumnName($x) . ' || ' . $this->helper->quoteColumnName($y) . ')');
 	}
 
+	public function groupConcat($expr, ?string $separator = ',', ?string $orderBy = null): IQueryFunction {
+		$separator = $this->helper->quoteColumnName($separator);
+		return new QueryFunction('GROUP_CONCAT(' . $this->helper->quoteColumnName($expr) . "$separator)");
```

Comment:
```suggestion
		return new QueryFunction('GROUP_CONCAT(' . $this->helper->quoteColumnName($expr) . ", $separator)");
```

Will add a real unit test with real data

Reply:
Since order is not supported here, should we drop it from the API?

Reply:
Order isn't mandatory. This function works fine without order on SQLite

Reply:
I saw now that the CI don't run tests using SQLite. i fixed the comma and did a `push -f`

Reply:
order removed.